source/license.md

```diff
@@ -1,6 +1,6 @@
-# <i icon-name="copyleft"></i> License
+# <i icon-name="copyleft"></i> Licence
```

Comment:
Are we sure about the British English way to write this word?

Reply:
https://www.oed.com/dictionary/licence_n?tab=factsheet#39414049
https://www.oed.com/dictionary/license_v?tab=factsheet#39415297
https://prowritingaid.com/licence-vs-license
"Usage—Note that in British English licence is the correct spelling for the noun and is also an acceptable variant spelling of the verb. In US English, both noun and verb are spelled license."

Reply:
I'm sorry, re-reading again through the Oxford dictionary I realised I misread the order of the greyed out “Am / Brit” information. Thank you for having checked better than me!

<img width="197" alt="image" src="https://github.com/openfisca/openfisca-doc/assets/222463/8440a925-db4b-4d73-804f-ab8fdcf77e4a">

And I'm glad that the British orthography aligns with the French one, this will make some translation work easier 🙂